src/components/comment-box.jsx

```diff
@@ -1,22 +1,74 @@
 import React, { PropTypes } from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
 
+import * as commentActions from '../actions/commentActions';
 import CommentForm from './comment-form';
 import CommentList from './comment-list';
 
-const CommentBox = (props) => (
-  <div className="commentBox">
-    <h1>Comments</h1>
-    <CommentList comments={props.comments} />
-    <CommentForm handleSubmit={props.handleSubmit} />
-  </div>
-);
-
-CommentBox.propTypes = {
-  comments: PropTypes.arrayOf(PropTypes.shape({
-    author: PropTypes.string,
-    message: PropTypes.string,
-  })),
-  handleSubmit: PropTypes.func.isRequired,
-};
-
-export default CommentBox;
+class CommentBox extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      comment: Object.assign({}, this.props.comment),
+      isSaving: false
+    };
+
+    this.updateComment = this.updateComment.bind(this);
```

Comment:
Since we are using `stage-2` and have access to class properties now, maybe omit these binds here and add them as arrow functions:

```js
class CommentBox {
  updateComment = () => {
    //...
  };
  
  saveComment = () => {
    //...
  };
}
```